### DISCUSSION TITLE
Get geolocation for photo, video from GoPro

### DISCUSSION THREAD

**Discussion by sultanmyrza** [gopro/OpenGoPro#57] [Q&A]
It's not document in official [docs](https://gopro.github.io/OpenGoPro/wifi#what-can-you-do-with-gopro-api) yet **but how we can get Geolocation info from GoPro's photos and videos?**

[Here is discussed on how to get metadata but those metadata info does not contain gelocation info.](https://github.com/gopro/OpenGoPro/discussions/55)

Also from [docs](https://gopro.github.io/OpenGoPro/wifi#commands-quick-reference) 
I tried `/gopro/media/info?path=100GOPRO/XXX.JPG` but 
<details>
  <summary>
  the response does not container gelocation info
  </summary>
  
  ```json
  {
      "url": "http://10.5.5.9:8080/gopro/media/info?path=100GOPRO/GOPR0001.JPG",
      "headers": {
          "Content-Type": "application/json",
          "Content-Length": "235"
      },
      "data": {
          "us": "0",
          "fov": "19",
          "wdr": "0",
          "eis": "0",
          "raw": "0",
          "rot": "0",
          "ct": "4",
          "mp": "0",
          "cre": "1628286547",
          "gumi": "0696696448aefcdb6a649135c9c23d74",
          "hdr": "0",
          "tr": "0",
          "prjn": "6",
          "hc": "0",
          "s": "5742426",
          "w": "5184",
          "lc": "0",
          "mos": [],
          "h": "3888"
      },
      "status": 200
  }
```
</details>

**Comment by KonradIT** [comment]
For videos you can use the GPMF datastream and parse the location.
